src/main/scala/uk/gov/nationalarchives/tdr/api/service/ValidateFileMetadataService.scala

```diff
@@ -5,6 +5,7 @@ import uk.gov.nationalarchives.Tables
 import uk.gov.nationalarchives.Tables.{FilemetadataRow, FilestatusRow}
```

Comment:
There are some unused imports that should be removed:

- `import uk.gov.nationalarchives`
- `import java.sql.Timestamp`